toolbox/test-graphviz.txt

```diff
@@ -0,0 +1,31 @@
+digraph regexp { 
```

Comment:
can you remove this file from the commit. It's an artifact of the server build. Should be in the gotignore